### DISCUSSION TITLE
Can I add DNS records to the existing example.github.io domain?

### DISCUSSION THREAD

**Discussion by LSMorty** [community/community#22828] [Other Feature Feedback, Questions, & Ideas]
<p>So I am running a test site, where I am experimenting with server-side tracking, and I need to send the tracking data to a sub-domain that I “own”. In that regard, i need to add some TXT, A, and AAAA records to a subdomain different than the <a href="http://example.github.io" rel="noopener nofollow ugc">example.github.io</a>.</p>
<p>I know that it is possible to configure with my own domain, but as this is just a test project, I have no purpose for setting up a custom domain and hereby endure costs.</p>
<p>So is there a way that I can access the existing domain’s DNS settings and create the necessary DNS records?</p>

**Comment by otter-computer** [comment, marked as answer]
<p>No, <code>github.io</code> domain DNS records are automatically managed by GitHub. There isn’t a way for you to modify the DNS records attached to it. You’ll need to use your own domain if you need to set up any DNS records.</p>

**Comment by LSMorty** [comment]
<p>Cheers <a class="mention" href="/thomasshaped">@thomasshaped</a>,</p>
<p>I will use my own domain.</p>
